files/chat.osuosl.org.signing.key

```diff
@@ -0,0 +1 @@
+ed25519 a_cDWo IgroQ8oS8U+uxtDmS8czW3fVnTO/qmdJL7EOboYz4Os
```

Comment:
Shouldn't this be a secret?

Reply:
I didn't intend on committing anything that was present in the `files/` directory, as it was only for testing purposes. The cookbook doesn't rely on any of these files.

Reply:
Can this be removed then?

Reply:
It was an oversight.